doc/source/dev/contributor/contributor_toc.rst

```diff
@@ -17,6 +17,7 @@ This guide is designed to help you quickly find the information you need about S
 
 Development environment
 -----------------------
+- :ref:`quickerstart-conda` contains just the commands you need to get started on Mac and Linux
```

Comment:
... on Mac and Linux with conda.

Reply:
Actually, is conda not cross-platform? I.e. will this all work on Windows? --- this can  however be left for a follow-up PR.

Reply:
Building has always been so tricky on Windows i didn't even think about it. I'll try if nobody knows why it wouldn't. @ilayn ?

Reply:
The [compilers package](https://anaconda.org/conda-forge/compilers) has `win-64` packages, so it may work.

Reply:
@egorz734 can you or Diana try these instructions on Windows?

Reply:
Re Windows: does conda take care of blas/lapack dependencies?  If not, a quick way to get openblas is to follow the CI steps (the steps of which are partially fleshed out [here](https://github.com/scipy/scipy/wiki/Easy-Windows-10-Build-Scipy-from-source#install-openblas) for the impatient)

Reply:
Yes, you can install compilers and all other builds dependencies with conda - the instructions in this PR do that.

Reply:
I have no idea about conda on windows unfortunately. Never used it.

Reply:
I tried these on Windows. `conda list` shows that `mkl` is installed, but I run into:
```
Running from SciPy source directory.
lapack_opt_info:
lapack_mkl_info:
No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
customize MSVCCompiler
  libraries mkl_rt not found in ['C:/ProgramData/Anaconda3/envs/scipy\\Library\\lib']
  NOT AVAILABLE

openblas_lapack_info:
  libraries openblas not found in ['C:/ProgramData/Anaconda3/envs/scipy\\Library\\lib']
Traceback (most recent call last):
  File "setup.py", line 605, in <module>
    setup_package()
  File "setup.py", line 601, in setup_package
    setup(**metadata)
  File "C:\ProgramData\Anaconda3\envs\scipy\lib\site-packages\numpy\distutils\core.py", line 135, in setup
    config = configuration()
  File "setup.py", line 502, in configuration
    lapack_opt = get_info('lapack_opt')
  File "C:\ProgramData\Anaconda3\envs\scipy\lib\site-packages\numpy\distutils\system_info.py", line 485, in get_info
    return cl().get_info(notfound_action)
  File "C:\ProgramData\Anaconda3\envs\scipy\lib\site-packages\numpy\distutils\system_info.py", line 746, in get_info
    self.calc_info()
  File "C:\ProgramData\Anaconda3\envs\scipy\lib\site-packages\numpy\distutils\system_info.py", line 1771, in calc_info
    if self._calc_info(lapack):
  File "C:\ProgramData\Anaconda3\envs\scipy\lib\site-packages\numpy\distutils\system_info.py", line 1748, in _calc_info
    return getattr(self, '_calc_info_{}'.format(name))()
  File "C:\ProgramData\Anaconda3\envs\scipy\lib\site-packages\numpy\distutils\system_info.py", line 1665, in _calc_info_openblas
    info = get_info('openblas_lapack')
  File "C:\ProgramData\Anaconda3\envs\scipy\lib\site-packages\numpy\distutils\system_info.py", line 485, in get_info
    return cl().get_info(notfound_action)
  File "C:\ProgramData\Anaconda3\envs\scipy\lib\site-packages\numpy\distutils\system_info.py", line 746, in get_info
    self.calc_info()
  File "C:\ProgramData\Anaconda3\envs\scipy\lib\site-packages\numpy\distutils\system_info.py", line 2123, in calc_info
    info = self._calc_info()
  File "C:\ProgramData\Anaconda3\envs\scipy\lib\site-packages\numpy\distutils\system_info.py", line 2094, in _calc_info
    f = new_fcompiler(c_compiler=c)
  File "C:\ProgramData\Anaconda3\envs\scipy\lib\site-packages\numpy\distutils\fcompiler\__init__.py", line 876, in new_fcompiler
    load_all_fcompiler_classes()
  File "C:\ProgramData\Anaconda3\envs\scipy\lib\site-packages\numpy\distutils\fcompiler\__init__.py", line 777, in load_all_fcompiler_classes
    __import__ (module_name)
  File "C:\ProgramData\Anaconda3\envs\scipy\lib\site-packages\numpy\distutils\fcompiler\compaq.py", line 54, in <module>
    class CompaqVisualFCompiler(FCompiler):
  File "C:\ProgramData\Anaconda3\envs\scipy\lib\site-packages\numpy\distutils\fcompiler\compaq.py", line 78, in CompaqVisualFCompiler
    m.initialize()
  File "C:\ProgramData\Anaconda3\envs\scipy\lib\site-packages\numpy\distutils\msvccompiler.py", line 52, in initialize
    os.environ['include'] = _merge(environ_include, os.environ['include'])
  File "C:\ProgramData\Anaconda3\envs\scipy\lib\os.py", line 675, in __getitem__
    raise KeyError(key) from None
KeyError: 'include'
```

I used conda to install openblas, and I run into:
```
Running from SciPy source directory.
lapack_opt_info:
lapack_mkl_info:
No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
customize MSVCCompiler
  libraries mkl_rt not found in ['C:/ProgramData/Anaconda3/envs/scipy\\Library\\lib']
  NOT AVAILABLE

openblas_lapack_info:
creating C:\Users\ADMINI~1\AppData\Local\Temp\2\tmpy5f3tkr2\Users
creating C:\Users\ADMINI~1\AppData\Local\Temp\2\tmpy5f3tkr2\Users\ADMINI~1
creating C:\Users\ADMINI~1\AppData\Local\Temp\2\tmpy5f3tkr2\Users\ADMINI~1\AppData
creating C:\Users\ADMINI~1\AppData\Local\Temp\2\tmpy5f3tkr2\Users\ADMINI~1\AppData\Local
creating C:\Users\ADMINI~1\AppData\Local\Temp\2\tmpy5f3tkr2\Users\ADMINI~1\AppData\Local\Temp
creating C:\Users\ADMINI~1\AppData\Local\Temp\2\tmpy5f3tkr2\Users\ADMINI~1\AppData\Local\Temp\2
creating C:\Users\ADMINI~1\AppData\Local\Temp\2\tmpy5f3tkr2\Users\ADMINI~1\AppData\Local\Temp\2\tmpy5f3tkr2
cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MD /TcC:\Users\ADMINI~1\AppData\Local\Temp\2\tmpy5f3tkr2\source.c /FoC:\Users\ADMINI~1\AppData\Local\Temp\2\tmpy5f3tkr2\Users\ADMINI~1\AppData\Local\Temp\2\tmpy5f3tkr2\source.obj
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\scipy\lib\distutils\_msvccompiler.py", line 405, in compile
    self.spawn(args)
  File "C:\ProgramData\Anaconda3\envs\scipy\lib\distutils\_msvccompiler.py", line 511, in spawn
    return super().spawn(cmd)
  File "C:\ProgramData\Anaconda3\envs\scipy\lib\site-packages\numpy\distutils\ccompiler.py", line 90, in <lambda>
    m = lambda self, *args, **kw: func(self, *args, **kw)
  File "C:\ProgramData\Anaconda3\envs\scipy\lib\site-packages\numpy\distutils\ccompiler.py", line 173, in CCompiler_spawn
    raise DistutilsExecError('Command "%s" failed with exit status %d%s' %
distutils.errors.DistutilsExecError: Command "cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MD /TcC:\Users\ADMINI~1\AppData\Local\Temp\2\tmpy5f3tkr2\source.c /FoC:\Users\ADMINI~1\AppData\Local\Temp\2\tmpy5f3tkr2\Users\ADMINI~1\AppData\Local\Temp\2\tmpy5f3tkr2\source.obj" failed with exit status 127

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "setup.py", line 605, in <module>
    setup_package()
  File "setup.py", line 601, in setup_package
    setup(**metadata)
  File "C:\ProgramData\Anaconda3\envs\scipy\lib\site-packages\numpy\distutils\core.py", line 135, in setup
    config = configuration()
  File "setup.py", line 502, in configuration
    lapack_opt = get_info('lapack_opt')
  File "C:\ProgramData\Anaconda3\envs\scipy\lib\site-packages\numpy\distutils\system_info.py", line 485, in get_info
    return cl().get_info(notfound_action)
  File "C:\ProgramData\Anaconda3\envs\scipy\lib\site-packages\numpy\distutils\system_info.py", line 746, in get_info
    self.calc_info()
  File "C:\ProgramData\Anaconda3\envs\scipy\lib\site-packages\numpy\distutils\system_info.py", line 1771, in calc_info
    if self._calc_info(lapack):
  File "C:\ProgramData\Anaconda3\envs\scipy\lib\site-packages\numpy\distutils\system_info.py", line 1748, in _calc_info
    return getattr(self, '_calc_info_{}'.format(name))()
  File "C:\ProgramData\Anaconda3\envs\scipy\lib\site-packages\numpy\distutils\system_info.py", line 1665, in _calc_info_openblas
    info = get_info('openblas_lapack')
  File "C:\ProgramData\Anaconda3\envs\scipy\lib\site-packages\numpy\distutils\system_info.py", line 485, in get_info
    return cl().get_info(notfound_action)
  File "C:\ProgramData\Anaconda3\envs\scipy\lib\site-packages\numpy\distutils\system_info.py", line 746, in get_info
    self.calc_info()
  File "C:\ProgramData\Anaconda3\envs\scipy\lib\site-packages\numpy\distutils\system_info.py", line 2123, in calc_info
    info = self._calc_info()
  File "C:\ProgramData\Anaconda3\envs\scipy\lib\site-packages\numpy\distutils\system_info.py", line 2111, in _calc_info
    if not (skip_symbol_check or self.check_symbols(info)):
  File "C:\ProgramData\Anaconda3\envs\scipy\lib\site-packages\numpy\distutils\system_info.py", line 2190, in check_symbols
    obj = c.compile([src], output_dir=tmpdir)
  File "C:\ProgramData\Anaconda3\envs\scipy\lib\distutils\_msvccompiler.py", line 407, in compile
    raise CompileError(msg)
distutils.errors.CompileError: Command "cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MD /TcC:\Users\ADMINI~1\AppData\Local\Temp\2\tmpy5f3tkr2\source.c /FoC:\Users\ADMINI~1\AppData\Local\Temp\2\tmpy5f3tkr2\Users\ADMINI~1\AppData\Local\Temp\2\tmpy5f3tkr2\source.obj" failed with exit status 127
```
I tried install build tools according to the Windows build guide, but I don't think that changed the error message.

Reply:
How about we merge this PR as is, and iterate on the windows quickstart part?

Reply:
Fine with me. Just thought I'd try it.

Reply:
+1 for merging.  Can do if everyone is happy?

Reply:
SGTM:)